oamapps/columnstoreSupport/logReport.sh

```diff
@@ -14,15 +14,11 @@ else
         INSTALLDIR="/usr/local/mariadb/columnstore"
 fi
```

Comment:
same comment applies to all of these scripts; we should get these from the env or the config file